include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IDEInstInteractionAnalysis.h

```diff
@@ -146,25 +146,48 @@ class IDEInstInteractionAnalysisT
       return std::make_shared<Gen<d_t>>(Alloca, this->getZeroValue());
     }
 
-    // // Handle indirect taints, i. e., propagate values that depend on branch
-    // // conditions whose operands are tainted.
-    // if (EnableIndirectTaints) {
-    //   if (auto br = llvm::dyn_cast<llvm::BranchInst>(curr);
-    //       br && br->isConditional()) {
-    //     return std::make_shared<LambdaFlow<d_t>>([=](d_t src) {
-    //       container_type ret = {src, br};
-    //       if (src == br->getCondition()) {
-    //         for (auto succ : br->successors()) {
-    //           // this->indirecrTaints[succ].insert(src);
-    //           for (auto &inst : succ->instructionsWithoutDebug()) {
-    //             ret.insert(&inst);
-    //           }
-    //         }
-    //       }
-    //       return ret;
-    //     });
-    //   }
-    // }
+    // Handle indirect taints, i. e., propagate values that depend on branch
+    // conditions whose operands are tainted.
+    if (EnableIndirectTaints) {
```

Comment:
```suggestion
    if constexpr (EnableIndirectTaints) {
```
Should not matter much but putting constexpr here makes it clear that EnableIndirectTaints is a compile time constant and the block will not be compiled in when it is `false`